src/main/scala/org/broadinstitute/dsde/firecloud/webservice/EntityApiService.scala

```diff
@@ -67,11 +68,11 @@ trait EntityApiService extends FireCloudDirectives
                 }
               } ~
               pathPrefix(Segment) { entityType =>
-                streamingPassthrough(baseRawlsEntitiesUrl + "/" + entityType)
+                streamingPassthrough(FireCloudConfig.Rawls.entityPathFromWorkspace(escapePathSegment(workspaceNamespace), escapePathSegment(workspaceName)) + "/" + entityType)
```

Comment:
I see that `baseRawlsEntitiesUrl` is used throughout this file - why does it only need to be replaced here?

Reply:
The other usages are working correctly (and have been working correctly for a while), because the old `passthrough` directive already has some encoding capabilities. The new `streamingPassthrough` requires the encoding in this PR.